src/test/java/com/github/princesslana/eriscasper/data/resource/TestUserWithGuildId.java

```diff
@@ -1,31 +0,0 @@
-package com.github.princesslana.eriscasper.data.resource;
-
-import com.github.princesslana.eriscasper.data.DataAssert;
-import com.github.princesslana.eriscasper.data.Snowflake;
-import java.util.Optional;
-import org.testng.annotations.Test;
-
-public class TestUserWithGuildId {
```

Comment:
Why is this test being removed?

Seems that it should deserialize to something else (so will need tweaking/renaming), but it still seems like something that should be a valid test.